ruby-gem/bin/configure

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+lib_path = File.expand_path('../lib', __dir__)
+$:.unshift(lib_path) if !$:.include?(lib_path)
+require 'configure'
+
+def help
+	puts <<~HELP
+		configure
+		A command-line utility for applying configuration secrets with strong encryption
+
+		USAGE:
+		bundle exec configure <SUBCOMMAND>
```

Comment:
Nitpick. I wonder if it's necessary/ strictly speaking correct to add `bundle exec` here? 🤔 It should work as a standalone command, right? Still, `bundle exec` is the _recommended_ way to run this...

```suggestion
		configure <SUBCOMMAND>
```

🤔 🤷‍♂️

Reply:
Yeah I agree, even if we highly recommend using any gem – this one included – as part of a `Gemfile` and via `bundler`, ultimately this is a ruby context detail. One could as well decide to install the `configure` gem system-wide (or `rbenv` ruby-version-wide) instead of using a `Gemfile`, and invoke `configure` directly (that would be their loss, but still totally valid). Or use `asdf`. Or have their `zsh` configured to auto-bundle gem shims (that's what I do).